include/kimera_vio_ros/KimeraVioNode.hpp

```diff
@@ -0,0 +1,67 @@
+// #include <chrono>
+#include <functional>
+#include <future>
+#include <memory>
+#include <string>
+#include <thread>
+
+// Still need gflags for parameters in VIO
+#include <gflags/gflags.h>
+#include <glog/logging.h>
+
+#include "image_transport/subscriber_filter.h"
+
+#include "message_filters/subscriber.h"
+#include "message_filters/synchronizer.h"
+#include "message_filters/sync_policies/exact_time.h"
+
+#include "rclcpp/rclcpp.hpp"
+
+#include "sensor_msgs/msg/image.hpp"
+#include "sensor_msgs/msg/imu.hpp"
+
+#include <kimera-vio/pipeline/Pipeline.h>
+#include "kimera_vio_ros/RosDataProviderInterface.hpp"
+
+
+using namespace message_filters::sync_policies;
+using namespace sensor_msgs::msg;
```

Comment:
Can we avoid `using namespace`?
Use an alias instead if it is just too long of a namespace:
```
namespace fbz = foo::bar::baz;
```

Reply:
Sure thing, I'll migrate to alias when I refactor.